src/SimpleGraphs/generators/staticgraphs.jl

```diff
@@ -763,3 +763,25 @@ function lollipop_graph(n1::T, n2::T) where {T <: Integer}
     add_edge!(g, n1, n1+1)
     return g
 end
+
+"""
+    friendship_graph(n)
+Create a [friendship graph](https://en.wikipedia.org/wiki/Friendship_graph) 
+consisting of n copies of the cycle graph C3 with a common vertex`.
+"""
+
+function friendship_graph(n::T) where {T <: Integer} 
+    n <= 0 && return SimpleGraph{T}(1)
+    g = SimpleGraph{T}(2 * n + 1)
+    indx = one(T)
+    while indx <= n
```

Comment:
It's probably more straightforward to do something like
```
nvg = 2n + 1
g = SimpleGraph(nvg)
for i = 2:2:nvg-1
    add_edge!(g, 1, i)
    add_edge!(g, 1, i+1)
    add_edge!(g, i, i+1)
end
```

That is, by the time we create the graph we shouldn't really need to know anything about `n`.

Reply:
I think of it a some kind of induction.

Reply:
You still could use something like `for indx in 1:n`.